clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -2136,15 +2136,12 @@ Address CodeGenFunction::EmitIntelFPGAFieldAnnotations(SourceLocation Location,
                                                        StringRef AnnotStr) {
   llvm::Value *V = Addr.getPointer();
   llvm::Type *VTy = V->getType();
+  assert(isa<llvm::PointerType>(VTy) &&
```

Comment:
Can you use `cast` and make `VTy` a `PointerType` instead of this assert?

Reply:
Well, I think I just don't need any of them.